src/apple/macos/ffi.rs

```diff
@@ -109,12 +112,16 @@ extern "C" {
 
 pub type DADiskRef = *const __DADisk;
 
-#[allow(non_camel_case_types)]
-pub type io_object_t = mach_port_t;
-#[allow(non_camel_case_types)]
-pub type io_connect_t = io_object_t;
-#[allow(non_camel_case_types)]
-pub type io_iterator_t = io_object_t;
+cfg_if! {
```

Comment:
Same as above.

Reply:
Without a `cfg_if! {}` block here and below, I would need to gate all the types and structures individually. Am I missing something else?

Reply:
You can put them in a `mod` (named the same for both) and then reexport all the `mod` items.

Reply:
Thanks! I got rid of all the`cfg_if` usages.